eng/deployment/postgresql-kafka-opensearch/README.md

```diff
@@ -0,0 +1,76 @@
+# PostgreSQL -> Kafka -> OpenSearch
+
+>⚠️  **WARNING: NOT FOR PRODUCTION USE!** Includes passwords in the default configuration that are visible in this repo and should never be used in real life. Be very careful!
+
+For requirements and design background, see [Change Data Capture to
+Stream](https://github.com/Ed-Fi-Alliance-OSS/Project-Tanager/blob/main/docs/DMS/CDC-STREAMING.md).
+
+1. Start these containers.
+
+    ```shell
+    docker compose up -d
+    ```
+
+1. Open a shell on the PostgreSQL container and run the following to enable
+   replication:
+
+    ```bash
+    echo "host    replication    postgres         kafka-postgresql-source    trust" >> /var/lib/postgresql/data/pg_hba.conf
+    echo "wal_level = logical" >> /var/lib/postgresql/data/postgresql.conf
+    ```
+
+1. Restart the PostgreSQL container.
+1. Create the `edfi_datamanagementservice` database and run the DB installer. 
+1. Run the following SQL commands against the `edfi_datamanagementservice` database:
+    ```sql
+    CREATE PUBLICATION to_debezium WITH (publish = 'insert, update, delete, truncate', publish_via_partition_root = true);
+    SELECT pg_create_logical_replication_slot('debezium', 'pgoutput');
+    ALTER PUBLICATION to_debezium ADD TABLE dms.document;
+    ```
+1. Run `setup.ps1` to inject the connector configurations into the respective
+   sink and source connector containers.
+1. Start the DMS and begin inserting and updating documents.
+1. View Kafka changes in Kafdrop and OpenSearch changes in its Dashboard
+
+Tools
+
+* [Kafdrop](http://localhost:9000/)
+* [OpenSearch Dashboard](http://localhost:5601/)
+  * After a record has been inserted, create an [index
+    pattern](http://localhost:5601/app/management/opensearch-dashboards/indexPatterns): `ed-fi$*`
+
+Known problems:
+
+* DELETE is not having any impact on OpenSearch.
```

Comment:
If we don't already have a ticket for that, please create one. I suspect that we're not emitting the tombstone event properly from our transform.